src/index.ts

```diff
@@ -1,29 +1,55 @@
 import { reactotron, reactotronWithRedux } from './reactotron';
```

Comment:
I would keep the index.ts light with just the import/export statements. The code could be moved to one or multiple files (it might be better to extract the logic of each feature (logger, crash, reactotron, ...) and put it in a separate file)